### PR TITLE
CDK updates

### PR DESCRIPTION
**Pull request by kmurphypolygon**
- Full review of content.

**TODOs:**
- Standardize placeholder image styles.
- Discuss issues with quickstart.
- Validium deploy step 3 outstanding.
- Question about what makes these products CDK rather than just the products themselves?
- Manage allowlists -> prly should be Manage policies.
- Performance benchmarks to do up-to-date results, conclusions, comparisons: https://polygon.atlassian.net/browse/TKDW-4
- That's it, I think, for now.

### CODE REVIEW

File: docs/cdk/concepts/dac.md [hunk 1]
```diff
@@ -1,15 +1,31 @@
-!!! note
-    fix this: https://medium.com/coinmonks/what-is-data-availability-and-why-do-we-need-it-7993c039fc44
+Data availability in blockchain technology refers to whether or not transaction data is available on-chain or off-chain. 
 
+If data is available on-chain, then each node has to store every piece of data related to every transaction. This can be costly for node operators as hardware and storage requirements are high. Moving data off-chain streamlines node architecture and operations.
 
-A data availability committee (DAC) is an optional offchain network of nodes that provides data to a blockchain network. The advantages are:
+!!! quote "Recommended resource"
```
Comment (cerberushades) [inline docs/cdk/concepts/dac.md]: I'd perhaps use !!! info instead

File: docs/cdk/concepts/dac.md [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
 
-- **Lower transaction fees**: Reduced computational requirements lead to lower fees.
-- **State privacy**: A secure copy of state transitions ensures data integrity and privacy.
+## What is a validium?
 
-DACs store the data required to reconstruct the state of the blockchain and make that data accessible so that, if the blockchain goes down, users can still access their assets and data.
+Validiums are blockchain solutions that do not store transaction data. 
 
-Delegating blockchain data to a DAC in this way can be costly. However, a DAC improves finality and thus supports Enterprise use cases which require cheap and fast transactions with a private and secure data layer.
+!!! quote "Recommended resource"
```
Comment (cerberushades) [inline docs/cdk/concepts/dac.md]: use !!! info ?

File: docs/cdk/concepts/dac.md [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+A DAC is an off-chain network of nodes that implements a data storage layer in a modular chain architecture like CDK. Some advantages are:
+
+- **Lower transaction fees**: Reduced computational requirements lead to lower fees and faster finality on the main network.
+- **Privacy and security**: The DAC stores secure and private transaction data.
+- **Trustless**: Data owners do not have to trust the blockchain operator.
+- **Scalable**: Decoupling data storage from the main chain supports more scalable networks. 
+
+DACs store the data required to reconstruct the state of the blockchain and make that data accessible so that, if the main blockchain network goes down, users can still access their assets and data.
+
+DAC set up can be costly and therefore DACs mostly support Enterprise use cases which require cheap and fast transactions with a private and secure data layer.
```
Comment (cerberushades) [inline docs/cdk/concepts/dac.md]:
```suggestion
Setting up a DAC can be costly and therefore DACs mostly support Enterprise use cases which require cheap and fast transactions with a private and secure data layer.
```
It still looks unclear to me. 
Maybe something like "Setting up a DAC can be costly and therefore DACs tend to be used to support Enterprise use cases which require cheap and fast transactions with a private and secure data layer."

File: docs/cdk/concepts/dac.md [hunk 4]
```diff
@@ -31,5 +47,4 @@ The DAC works together with the sequencer to control the flow of data. The proce
 
 6. **Verification on Ethereum**: A designated smart contract on Ethereum verifies the submitted signatures against a list of valid DAC members and confirms that sufficient approval has been provided for the batch hash.
 
-7. **Final settlement with zero-knowledge proof**: The aggregator prepares a proof for the batch via the prover and submits it to Ethereum. This proof confirms the validity of the batch's transactions without revealing their details. The chain state updates on Ethereum.
-
+7. **Final settlement with zero-knowledge proof**: The aggregator prepares a proof for the batch via the prover and submits it to Ethereum. This proof confirms the validity of the batch's transactions without revealing their details. The chain state updates on Ethereum.
```
Comment (cerberushades) [inline docs/cdk/concepts/dac.md:50]:
```suggestion
7. **Final settlement with zero-knowledge proof**: The aggregator prepares a proof for the batch via the prover and submits it to Ethereum. This proof confirms the validity of the batch's transactions without revealing their details. The chain state gets updated on Ethereum.
```

File: docs/cdk/get-started/deploy-rollup.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+!!! important "Recommendation"
```
Comment (cerberushades) [inline docs/cdk/get-started/deploy-rollup.md]: although this note is being corrrectly rendered ,I don1t see tihs type of admonition here https://squidfunk.github.io/mkdocs-material/reference/admonitions/. It seems to be the same of !!! note

File: docs/cdk/get-started/deploy-rollup.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+    Follow the [quickstart](quickstart-rollup.md) for a hands-on introduction to CDK in rollup mode.
+
+## Deploy a CDK rollup locally
+
+The [zkEVM deployment quickstart documentation](../..//zkEVM/step1-fullzkevm.md) shows you how to deploy a zkEVM rollup on your local machine.
+
+!!! question
+    What makes this CDK?
+
```
Comment (cerberushades) [inline docs/cdk/get-started/deploy-rollup.md:10]: What is this question for?

File: docs/cdk/get-started/deploy-validium.md [hunk 7]
```diff
@@ -21,4 +21,7 @@ Follow the instructions in the [CDK validium node repository's README](https://g
 
     Finally, once the CDK validium node is operational, set up and run the data availability node. 
 
-    Instructions for this can be found in the [<ins>CDK DA Node GitHub repository's README</ins>](https://github.com/0xPolygon/cdk-data-availability).
\ No newline at end of file
+    Instructions for this can be found in the [<ins>CDK DA Node GitHub repository's README</ins>](https://github.com/0xPolygon/cdk-data-availability).
+
+!!! question
+    What makes this CDK?
```
Comment (cerberushades) [inline docs/cdk/get-started/deploy-validium.md:27]: ?

File: docs/cdk/get-started/quickstart-validium.md [hunk 8]
```diff
@@ -159,6 +159,9 @@ You should see a page similar to this:
   <img src="/img/cdk/cdk-metamask-import-account.png" alt="bridge" width="90%" height="30%" />
 </div>
 
+!!! danger
+    Today, 21/11, I got stuck here with pending transfers that weren't picked up by blockexplorer.
+
```
Comment (cerberushades) [inline docs/cdk/get-started/quickstart-validium.md:164]: I think we need to remove this.

File: docs/cdk/get-started/quickstart-validium.md [hunk 9]
```diff
@@ -240,12 +243,15 @@ The L2->L1 bridging is slightly different than L1->L2 and you will see the **Act
...
 
 <div align="center">
   <img src="/img/cdk/cdk-bridge-l2-to-l1-completed.png" alt="bridge" width="90%" height="30%" />
 </div>
 
 !!! urgent
     - First time, I had an error on step 5.12 -> cannot estimate gas.
-    - Second time, I had an error much earlier on, but that's probably because I didn't clear the images up, even though I restarted. 
\ No newline at end of file
+    - Second time, I had an error much earlier on, but that's probably because I didn't clear the images up, even though I restarted.
```
Comment (cerberushades) [inline docs/cdk/get-started/quickstart-validium.md]: to be removed

**Review by cerberushades — CHANGES_REQUESTED** [review]

File: docs/cdk/concepts/dac.md (hunk 1 above)
Comment (kmurphypolygon) [reply docs/cdk/concepts/dac.md]: agree

File: docs/cdk/concepts/dac.md (hunk 4 above)
Comment (kmurphypolygon) [reply docs/cdk/concepts/dac.md:50]: disagree, active versus passive

File: docs/cdk/get-started/deploy-validium.md (hunk 7 above)
Comment (kmurphypolygon) [reply docs/cdk/get-started/deploy-validium.md:27]: question for Gabriel later today

File: docs/cdk/get-started/deploy-rollup.md (hunk 6 above)
Comment (kmurphypolygon) [reply docs/cdk/get-started/deploy-rollup.md:10]: question for Gabriel later today

File: docs/cdk/get-started/deploy-rollup.md (hunk 5 above)
Comment (kmurphypolygon) [reply docs/cdk/get-started/deploy-rollup.md]: you're right, i'm making stuff up ... will change to note

File: docs/cdk/get-started/quickstart-validium.md (hunk 8 above)
Comment (kmurphypolygon) [reply docs/cdk/get-started/quickstart-validium.md:164]: for my chat with Gabriel

File: docs/cdk/get-started/quickstart-validium.md (hunk 9 above)
Comment (kmurphypolygon) [reply docs/cdk/get-started/quickstart-validium.md]: will remove all these later

**Comment by kmurphypolygon** [comment]
Hi @cerberushades all changes made and thank you :) ... (there are still todos of course, I'm meeting Gabriel later this evening while we incrementally fix things up) .. Could you give me the OK please?

**Review by cerberushades — APPROVED** [review]

**Pull request merged**